### PR TITLE
SOLR-15920 ReleaseWizard - Upgrade 'consolemenu' dependency to v0.7.1

### PR DESCRIPTION
**Pull request by janhoy**
https://issues.apache.org/jira/browse/SOLR-15920

Basically get rid of custom code which is now part of upstream 'consolemenu.

### CODE REVIEW

**Comment by janhoy** [comment]
@HoustonPutman I think this is good to go, but would appreciate a quick check. 92 lines deleted :)

**Review by risdenk — APPROVED** [review]

**Review by HoustonPutman — APPROVED** [review]
Tried it out and it looks good to me.

Though I did have to make the same change that Ishan did to make it work (`Holidays.Englad` -> `Holidays.UK`).

**Comment by janhoy** [comment]
That’s strange since the holiday rep was not changed. Unless a new dependency upgrade triggered a lib upgrade with this change. The `requirements.txt` should perhaps lock major version of dependencies instead of specifying minimum version.

**Comment by janhoy** [comment]
Mystery solved. We have pip requirement `holidays>=0.9.10` and in `0.15` released this autust they [removed `England`](https://github.com/dr-prodigy/python-holidays/releases/tag/v.0.15)

> Drop support for UK subdivisions as countries (England, Scotland..) (dr-p)

I'll make the change to UK and pin version to 0.15

**Review by risdenk — APPROVED** [review]

**Comment by janhoy** [comment]
I see Ishan already did the `UK` change, thanks. So committed a change to `requirements.txt` to pin all lib versions to avoid future surprises.

File: dev-tools/scripts/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-jira
\ No newline at end of file
+six==1.16.0
+Jinja2==3.1.1
+PyYAML==6.0
+holidays==0.16
+ics==0.7.2
+console-menu==0.7.1
+PyGithub==1.56
+jira==3.4.1
```
Comment (janhoy) [inline dev-tools/scripts/requirements.txt:8]: I pinned every dependency here. 
I releaize that these are shared between release wizard and the other py tools in this folder, so I cannot guarantee that all the other tools work with the pinned version. I tested `githubPRs.py` and it works.

Comment (janhoy) [reply dev-tools/scripts/requirements.txt]: Changed these to [`~=` Compatible Release](https://peps.python.org/pep-0440/#compatible-release) to allow bugfix upgrades.

**Review by risdenk — APPROVED** [review]

**Pull request merged**